pyproject.toml

```diff
@@ -5,8 +5,10 @@ version = "0.0.1"
 dependencies = [
     "django",
     "psycopg2",
+    "pre-commit",
+    "coverage",
```

Comment:
I think these two are `optional dependencies` not project dependencies. If you agree, can we please move these to another section ? 

reference: https://peps.python.org/pep-0631/

Reply:
I see what you mean. It's for testing not running the app itself. I'll move them

Reply:
I moved them to optional depencies under the name dev